src/quo2/components/settings/settings_item/component_spec.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.settings.settings-item.component-spec
+  (:require [quo2.components.settings.settings-item.view :as settings-item]
+            [test-helpers.component :as h]))
+
+(def props
```

Comment:
args for tests should ideally be the bare minimum for the test to work imo.

Reply:
trimmed it down a little